src/AdminTab/TableList.tsx

```diff
@@ -40,10 +34,9 @@ type TableDropdownProps = {
   showMetabaseHint: boolean;
 };
 
-const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMetabaseTab, showMetabaseHint }) => {
+const TableActions: FunctionComponent<TableDropdownProps> = ({ table, onOpenMetabaseTab, showMetabaseHint }) => {
```

Comment:
ah I missed this earlier - TableActionsProps now

Reply:
Thanks!